app/resources/create.html.template

```diff
@@ -723,7 +723,7 @@
         </div>
 
         {% ifequal params.role "provide" %}
-          {% include "add_note_base.html.template" %}
+          {% include "add_note_base.html.template" with enable_recaptcha=True %}
```

Comment:
Can you handle the case where params.role is not "provide"? It looks Captcha is not shown in that case. This is the case when you first click "I'm looking for someone", search for some name and click [Create new record] at the bottom.

Reply:
Done. Added reCAPTCHA in the other branch (sorry my oversight).
![image](https://user-images.githubusercontent.com/43229670/133543420-683e299b-cbc4-49b3-a47c-621a73c4c33f.png)